src/pages/blog.js

```diff
@@ -19,24 +23,52 @@ const BlogPage = ({
     const posts = [...edges, ...parsedMdxData] // Merge MDX and MD data into one array
         .filter((edge) => !!edge.node.frontmatter.date)
         .sort((a, b) => new Date(b.node.frontmatter.date) - new Date(a.node.frontmatter.date)) // Resort based on dates following merge
-        .map((edge) => <PostCard key={edge.node.id} post={edge.node} />)
+
+    const latestPost = <PostCard key={posts[0].node.id} post={posts[0].node} featured={true} />
+    const nonLatestPosts = posts.slice(1, -1).map((edge) => <PostCard key={edge.node.id} post={edge.node} />)
```

Comment:
If the latest post is index 0, why are we also removing the last item instead of just `slice(1)`?